internal/tiltfile/tiltfile_state.go

```diff
@@ -757,9 +767,58 @@ func (s *tiltfileState) checkForImpossibleLiveUpdates(m model.Manifest) {
 				"is a feature you need, let us know!", iTarg.DeploymentRef.String()))
 
 			// Only emit the warning once
-			return
+			return nil
+		}
+
+		err = s.validatedLiveUpdate(iTarg, g)
+		if err != nil {
+			return err
 		}
 	}
+	return nil
+}
+
+func (s *tiltfileState) validatedLiveUpdate(iTarget model.ImageTarget, g model.TargetGraph) error {
```

Comment:
now that this isn't a getter (😅), should be `validateLiveUpdate`

Reply:
done